src/Microsoft.Azure.Relay/HybridConnectionListener.cs

```diff
@@ -683,8 +695,8 @@ Task<WebSocket> EnsureConnectTask(CancellationToken cancellationToken)
             async Task<WebSocket> ConnectAsync(CancellationToken cancellationToken)
             {
                 this.listener.ThrowIfDisposed();
-
-                var webSocket = ClientWebSocketFactory.Create(this.listener.UseBuiltInClientWebSocket);
+                var webSocket = ClientWebSocketFactory.Create(this.listener.UseBuiltInClientWebSocket,
+                    this.listener.CustomClientWebSocketFactory);
```

Comment:
Style: All parameters on a single line or all on their own line:

Use
ClientWebSocketFactory.Create(
    this.listener.UseBuiltInClientWebSocket, this.listener.CustomClientWebSocketFactory);

or 

ClientWebSocketFactory.Create(
    this.listener.UseBuiltInClientWebSocket,
    this.listener.CustomClientWebSocketFactory);

Reply:
done